assets/js/app/editor/Components/Select.vue

```diff
@@ -7,7 +7,7 @@
             :limit="1000"
             :multiple="multiple"
             :options="options"
-            :searchable="searchable || taggable"
+            :searchable="taggable"
```

Comment:
Hi @LordSimal , first of all thanks for the great detailed PRs. It's a pleasure to read through the explanations and code! Keep up the awesome work!

This is something that was changed recently, to make Relations searchable. I assume we need to rebase? Or, alternatively, is it GitHub not updating the file changes after a rebase?

Reply:
Just refreshed the branch with current master but had to add that manually since some of my commits had changes in there.

Reply:
Nice, thanks!